src/pages/ServiceGraph/SummaryPanelNode.tsx

```diff
@@ -170,7 +170,7 @@ export default class SummaryPanelNode extends React.Component<SummaryPanelPropTy
     );
   };
 
-  private renderBadgeSummary = (hasCB: string, hasRR: string) => {
+  private renderBadgeSummary = (hasCB: string, hasRR: string, hasMissingSC: boolean) => {
```

Comment:
Not a blocker but not sure why 'hasMissingSC' didn't follow the same pattern as the other flag fields.

Reply:
that is confusing.

The "string" is there (as opposed to "boolean") becaise IIRC we support "unknown" as a value (to let someone reading the metadata know that it could not be determined if the value is true or false).

I would recommend just keeping things consistent, otherwise, it will be confusing in the future to see different types.

Reply:
It comes as a boolean from the API, so no need to check for strings.
@jmazzitelli Ok, changed it to receive a string. However, I had to do `.toString()` in the caller.